src/main/java/me/xmrvizzy/skyblocker/skyblock/shortcut/Shortcuts.java

```diff
@@ -0,0 +1,205 @@
+        commands.put("/ca", "/chat all");
+        commands.put("/cp", "/chat party");
+        commands.put("/cg", "/chat guild");
+        commands.put("/co", "/chat officer");
+        commands.put("/cc", "/chat coop");
+
+        // Message
+        commandArgs.put("/m", "/msg");
+
+        // Party
+        commandArgs.put("/pa", "/p accept");
+        commands.put("/pv", "/p leave");
+        commands.put("/pd", "/p disband");
+        commands.put("/rp", "/reparty");
+
+        // Visit
+        commandArgs.put("/v", "/visit");
+        commands.put("/vp", "/visit portalhub");
+    }
+
+    private static void registerMoreDefaultShortcuts() {
+        // Combat
+        commands.put("/spider", "/warp spider");
+        commands.put("/crimson", "/warp nether");
+        commands.put("/end", "/warp end");
+
+        // Mining
+        commands.put("/gold", "/warp gold");
+        commands.put("/cavern", "/warp deep");
+        commands.put("/dwarven", "/warp mines");
+        commands.put("/fo", "/warp forge");
+        commands.put("/ch", "/warp crystals");
+
+        // Foraging & Farming
+        commands.put("/park", "/warp park");
+        commands.put("/barn", "/warp barn");
+        commands.put("/desert", "/warp desert");
+        commands.put("/ga", "/warp garden");
+
+        // Other warps
+        commands.put("/castle", "/warp castle");
+        commands.put("/museum", "/warp museum");
+        commands.put("/da", "/warp da");
+        commands.put("/crypt", "/warp crypt");
+        commands.put("/nest", "/warp nest");
+        commands.put("/magma", "/warp magma");
+        commands.put("/void", "/warp void");
+        commands.put("/drag", "/warp drag");
+        commands.put("/jungle", "/warp jungle");
+        commands.put("/howl", "/warp howl");
+    }
+
+    protected static void saveShortcuts(MinecraftClient client) {
+        JsonObject shortcutsJson = new JsonObject();
+        shortcutsJson.add("commands", SkyblockerMod.GSON.toJsonTree(commands));
+        shortcutsJson.add("commandArgs", SkyblockerMod.GSON.toJsonTree(commandArgs));
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(SHORTCUTS_FILE))) {
+            SkyblockerMod.GSON.toJson(shortcutsJson, writer);
+            LOGGER.info("[Skyblocker] Saved {} command shortcuts and {} command argument shortcuts", commands.size(), commandArgs.size());
+        } catch (IOException e) {
+            LOGGER.error("[Skyblocker] Failed to save shortcuts file", e);
+        }
+    }
+
+    private static void registerCommands(CommandDispatcher<FabricClientCommandSource> dispatcher, CommandRegistryAccess registryAccess) {
+        for (String key : commands.keySet()) {
+            if (key.startsWith("/")) {
+                dispatcher.register(literal(key.substring(1)));
+            }
+        }
+        for (String key : commandArgs.keySet()) {
+            if (key.startsWith("/")) {
+                dispatcher.register(literal(key.substring(1)));
```

Comment:
If you want you could add a string parameter here so that when typing the command it doesn't show up as an error.